### PR TITLE
Changing to use variables in LESS

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do?
- Moves basic colours out into a variables LESS file
- Changes the location of all favicon and manifest file locations
- Indents bullet points

#### What unit or integration tests does this PR have?
N/A

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Check that the code seems sane
`npm install`
`grunt docs` will build the docs for you to view changes

#### How should this be manually tested?
- Run `npm install`
- Run `grunt docs` & check the example http://localhost:4000
- Ensure the Favicon appear at the top of your tab.
- Check links on homepage to ensure a range of Favicons are visible.

#### Any background context you want to provide?
Favicons are standard for all websites so we need to ensure they work perfectly.

#### What are the relevant tickets?
N/A

#### What gif best describes how you feel about this work?
high five

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've checked there is appropriate selenium test coverage.
- [x] I've included regression tests for any bug fixes.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile Android, Safari, Chrome for any visual changes.
- [x] I've tested this on desktop IE9/10/11, Firefox, Chrome for any visual changes.
- [x] I've checked any significant visual changes with a member of the UX/UI pod.
- [x] I've checked this work against the requirements of the Jira.
- [x] I've added and updated translations for all supported languages
- [x] I've informed those who need to know of any unforeseen impact of this work (database migrations, deployment procedure changes).
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this

#### IP or Developer Review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve run all the tests locally and they pass.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by GeorgeF72** [comment]
Taking Se/Dev

**Comment by LouisPryer** [comment]
Taking IP/Dev

**Comment by GeorgeF72** [comment]
All seems good to me :+1:

**Comment by LouisPryer** [comment]
Looks good :+1:

**Comment by shackpank** [comment]
On SE/Guru

File: docs/assets/favicons/manifest.json [hunk 1]
```diff
@@ -2,43 +2,43 @@
 	"name": "UI Toolkit",
```
Comment (shackpank) [inline docs/assets/favicons/manifest.json]: Doesn't look like you introduced, but this file mixes tabs & spaces...

**Comment by shackpank** [comment]
It's a little annoying that grunt task starting everything concurrently, it tried to open `localhost:4000` before starting a server to listen on that port, also not introduced here though...

+1

**Comment by jodiedoubleday** [comment]
I agree we know thats annoying, we will solves that shortly.

**Pull request merged**